.env.example

```diff
@@ -0,0 +1,14 @@
+# Required
+REACT_APP_TINK_CLIENT_ID=
+TINK_CLIENT_SECRET=
+
+# Optional overrides
+## used only in the client
+REACT_APP_TINK_LINK_BASEURL=
+
+## used only in the server
+TINK_DELEGATED_CLIENT_ID=
+TINK_API_URL=
+
+# Required only for payments
+REACT_APP_PAYMENT_REQUEST_ID='<PAYMENT_REQUEST_ID>'
```

Comment:
Could also add an optional env variable for tink link session? 
`REACT_APP_TINK_LINK_SESSION_ID`